easylist/easylist_adservers.txt

```diff
@@ -2417,6 +2417,7 @@
 ||bostonwall.com^
 ||bostoppa.club^
 ||bot-checker.com^
+||botanyjustify.com^
```

Comment:
Can be replaced with the regex rule (https://mypdns.org/my-privacy-dns/matrix/-/issues/3459)

```css
/^https?://([a-z0-9]{8,10}.)?[a-z0-9]{5,14}.(com|bid|online|top|club)/([a-z0-9]{2}/){3}[a-f0-9]{32}.js$/$script,3p
```